app/templates/base.html

```diff
@@ -20,6 +20,29 @@
     <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/styles.css') }}" />
 </head>
 
+    <body bgcolor="LightGray">
+    {% if title == 'Log In' %}
+    {% elif title == 'Sign Up' %}
+    {% elif title == 'Start' %}
+    {% elif title == 'Log Out'%}
+    {% else %}
+        <div>
+            Song Recommender:
+            <a href = "{{ url_for('home') }}">Home</a>
+            <a href = "{{ url_for('logout') }}">Logout</a>
+            <a href = "{{ url_for('account') }}">Account</a>
+        </div>
+    {% endif %}
+    <hr>
+        {% with messages = get_flashed_messages() %}
+        {% if messages %}
+        <ul>
+            {% for message in messages %}
+            <li>{{ message }}</li>
+            {% endfor %}
+        </ul>
+        {% endif %}
+        {% endwith %}
```

Comment:
This code (lines 23-45) should not be here since it was updated in a previous commit. That is, this entire block was updated here in this commit: db7a3ec01010b1dad8a6a319796061dfc9bdb980